aws/chef-bootstrap.sh

```diff
@@ -106,13 +107,13 @@ if [ -f /etc/chef/client.pem ] ; then
 fi
 
 if [ "${LOCAL_MODE}" = "1" ]; then
-  mkdir -p /opt/chef-zero/{cookbooks,environments}
+  mkdir -p /var/chef/{cookbooks,environments}
```

Comment:
hah, that pattern is familiar from my time spent in chef land. Makes sense to me!